src/gui/gtk.h

```diff
@@ -342,6 +342,7 @@ static inline void dt_ui_section_label_set(GtkWidget *label)
   gtk_widget_set_halign(label, GTK_ALIGN_FILL); // make it span the whole available width
   g_object_set(G_OBJECT(label), "xalign", 0.0, (gchar *)0);    // make the text left aligned
   gtk_widget_set_name(label, "section_label"); // make sure that we can style these easily
+  gtk_label_set_ellipsize(GTK_LABEL(label), PANGO_ELLIPSIZE_END); // ellipsize labels
```

Comment:
This is already in #5691; you may want to be careful when rebasing.

Reply:
damnit. thx!